src/cz/cvut/cognitive/override/DistractionScreenGUIController.java

```diff
@@ -365,6 +439,75 @@ public void clickStartButton()
                 
 	}
         
+          public void clickDefButton()
+	{
+            if (!defClick){
+                
+                resetOptions();
+                
+                CheckBox_collect.check();
+                Slider_collect.setValue(30);
+                DistractionSettings.setProbabilityCollect(30);
+                DistractionSettings.setCollect(true);
+                
+                CheckBox_rain.check();
+                Slider_rain.setValue(20);
+                DistractionSettings.setIntensityRain(20);
+                DistractionSettings.setRain(true);
+                
+                CheckBox_box.check();
+                Slider_box.setValue(30);
+                DistractionSettings.setProbabilityBox(20);
+                DistractionSettings.setBox(true);
```

Comment:
This is how i assign the default values. first i reset all options and then set them to these values. Will try to work it from txt file